lib/src/states/camera_picker_state.dart

```diff
@@ -369,9 +369,14 @@ class CameraPickerState extends State<CameraPicker>
 
   /// The method to switch between flash modes.
   /// 切换闪光灯模式的方法
-  Future<void> switchFlashesMode() async {
+  Future<void> switchFlashesMode(
+      {
+        FlashMode? currentFlashMode,
+        int infiniteLoopControl = 5,
```

Comment:
This should not be infinite. Instead, every mode can be mapped and stored correspondingly whether they're available with the current controller.

Reply:
Hi @AlexV525 ,

Thanks a lot for your quick review.

Is there a better way to know whether a `FlashMode` is available or not than trying it?

If not, we can still store each `FlashMode` availability for the controller using the existing method.
There are several options for storing:
1. Store only for the execution time in a member of `CameraPickerState` as a `List<FlashMode> validFlashModes`.
2. Store it on the device with `shared_preferences` plugin for example.
3. Store it somewhere else so that it can be shared with other identical devices (to be relevant, this would require more knowledge of the device through something like `device_info_plus` for example).

I believe option 1. is good enough and a lot more reasonable compared to what the others would bring.

What do you think @AlexV525 ?

Reply:
> Store only for the execution time in a member of `CameraPickerState` as a `List<FlashMode> validFlashModes`.

I was about to implement this way with `Map<CameraDescription, List<FlashMode>> invalidFlashModes`.

Reply:
I just tried what you mentioned. I hope this is what you had in mind. It works on my device.

Sorry, as I don't write or read Chinese, I had to Google-translate the comments. I hope they are fine.